### DISCUSSION TITLE
无法访问youtube

### DISCUSSION THREAD

**Discussion by zhubingge** [getlantern/lantern-forum-cn#499] [General]
无法访问youtube，已经通过lantern反馈问题，问题代码72301371
@Mike777Khoo 麻烦处理一下，谢谢。

**Comment by MaxMKMax** [comment]
@zhubingge 好的. 谢谢您的提交报告. 这会帮助我们很多. 我帮您反馈给科技部门!

**Reply by zhubingge** [reply]
@Mike777Khoo 有定位到问题吗？大概什么时候可以修复呢？我这边有什么可以做的吗？
一直连不上youtube啊。感觉速度也没有之前好了。

**Reply by MaxMKMax** [reply]
@zhubingge 您好. 已经帮您提交给技术部门了. 等待他们修复.
